package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "babel-node index.js",
     "dev:start": "babel-watch index.js",
-    "test": "mocha --require babel-polyfill --require babel-register test/*.js"
+    "test": "nyc --reporter=html --reporter=text mocha --require babel-polyfill --require babel-register test/*.js",
```

Comment:
I think this is all you need for the test report to show:

`nyc mocha --require babel-register`